server/passport/facebook.js

```diff
@@ -0,0 +1,57 @@
+var FacebookStrategy = require('passport-facebook').Strategy
+require('dotenv').config({ path: '../database.env' })
+const passport = require('passport')
+const database = require('../database_init')
+const { hash, compare } = require('../utils')
+
+/**
+ * Strategy options needed by passport
+ */
+const options = {
+  clientID: process.env.FACEBOOK_CLIENT_ID,
+  clientSecret: process.env.FACEBOOK_CLIENT_SECRET,
+  callbackURL: 'http://localhost:8080/api/login/facebookCallBack',
```

Comment:
Maybe add a comment to not forget it is a MVP hyperlink (localhost:8080 may change)

Reply:
Good idea